src/rules/appery.js

```diff
@@ -12,7 +12,9 @@
  *
  */
 
-function (user, context, callback) {
+function generateApperySessionToken(user, context, callback) {
+  const request = require('request');
```

Comment:
It's probably a long shot, but any chance we can find a replacement for `request`, which has been _fully  deprecated_ as of Feb 11, 2020: https://github.com/request/request/issues/3142

Reply:
Probably should think about this separately - sorry for the noise.